src/main/java/pl/put/poznan/madness/rest/SortingMadnessController.java

```diff
@@ -25,6 +28,19 @@ public class SortingMadnessController {
   @Resource
   private ISortRunner runner;
 
+  /**
+   * Accepts a POST request with a JSON body containing the sorting algorithms and data to be sorted.
+   * Validates the input and runs a benchmark on the provided algorithms using the given data.
+   * Example valid input data:
+   * <pre>{@code {
+   *   "algorithms": ["BUBBLE_SORT", "QUICK_SORT"],
+   *   "data": [3, 2, 5, 1, 4]
+   * }}</pre>
+   *
+   * @param sortInput a {@link pl.put.poznan.madness.rest.models.SortDto} object containing the algorithms and data to be sorted
+   * @return a {@link pl.put.poznan.madness.logic.models.SortBenchmarkResult SortBenchmarkResult} object representing the results of the benchmark
```

Comment:
W niektórych linkach masz cały namespace do klasy a w niektórych nie, wydaje mi się, że wszędzie powinny być pełne ścieżki, żeby to działało dobrze, ale popraw mnie jeśli się mylę

Reply:
Javadoc jest w stanie przetrawić oba sposoby i nawet tak samo je formatuje. For the sake of simplicity zmienię wszystkie zapisy na bez namespace